include/dlaf/permutations/general/impl.h

```diff
@@ -192,6 +195,81 @@ void Permutations<B, D, T, C>::call(const SizeType i_begin, const SizeType i_end
   }
 }
 
+template <class T, Coord C>
+void permuteJustLocal(const SizeType i_begin, const SizeType i_end,
```

Comment:
Should I add (at least a basic) doc also for this, or should I just skip this since we are going to tackle it in a different way soon™️?

Reply:
No need